src/python/pants/backend/javascript/subsystems/nodejs.py

```diff
@@ -133,6 +142,8 @@ class EnvironmentAware(Subsystem.EnvironmentAware):
 
                 * `{AsdfPathString.STANDARD}`, {AsdfPathString.STANDARD.description("Node.js")}
                 * `{AsdfPathString.LOCAL}`, {AsdfPathString.LOCAL.description("binaries")}
+                * `<NVM>`, all NodeJS versions under $NVM_DIR/versions/node
+                * `<NVM_LOCAL>`, the nvm installation with the version in BUILD_ROOT/.nvmrc
```

Comment:
Should it be possible to defer the version option to this file as well? Repeating the version in both `pants.toml` and `.nvmrc` (or `.node-version`, for that matter) seem like bad UX.

Reply:
Yep, infer. I think I'll leave this as is for now if that's alright, validating the contents should probably happen in all backends with asdf support for example, and I think that work is a larger tangle.